vocabularies/Hierarchy.json

```diff
@@ -86,6 +86,26 @@
         "$Type": "Edm.PropertyPath",
         "$Nullable": true,
         "@Core.Description": "Integer property of type `Edm.Int64` holding the the number of [matching](#MatchCount) descendants a node has in the unlimited hierarchy"
+      },
+      "ChangeNextSiblingAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that moves a node among its siblings, following [this template](#Template_ChangeNextSiblingAction)"
+      },
+      "ChangePreviousSiblingAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that moves a node before a newly-created sibling, following [this template](#Template_ChangePreviousSiblingAction)"
+      },
+      "InsertAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that inserts a root into a recursive hierarchy, following [this template](#Template_InsertAction)"
```

Comment:
Discussion on 30.05.23: Should it not just insert the entity into the hierarchy? It will be a root if it does not have a parent.